front/src/Phaser/Login/LogincScene.ts

```diff
@@ -132,4 +152,16 @@ export class LogincScene extends Phaser.Scene implements GameSceneInterface {
     shareUserPosition(UsersPosition: import("../../Connexion").MessageUserPositionInterface[]): void {
         throw new Error("Method not implemented.");
     }
+
+    deleteGroup(groupId: string): void {
+        throw new Error("Method not implemented.");
+    }
+
+    shareGroupPosition(groupPositionMessage: GroupCreatedUpdatedMessageInterface): void {
+        throw new Error("Method not implemented.");
```

Comment:
Why do we need this? This is because of an interface, right? Is it logical that LogincScene implements this interface?

Reply:
I'm not sure, but we have `GameSceneInterface` and it should represent all of them. For me, it's not a problem, and we could imagine a map loading or what ever can to use a function in this interface.